cdc/sink/codec/avro.go

```diff
@@ -173,6 +184,18 @@ func (a *AvroEventBatchEncoder) avroEncode(table *model.TableName, tableVersion
 	}, nil
 }
 
+func extractColumnInfos(cols []*model.Column) []*model.ColumnInfo {
+	ret := make([]*model.ColumnInfo, 0)
```

Comment:
```suggestion
	ret := make([]*model.ColumnInfo, 0, len(cols))
```

Reply:
This function will be abandoned in the next PR (not submitted yet).